### ISSUE TITLE
Can't resolve 'react-art-svg-renderer' 

### ISSUE THREAD

**Issue by lakaniemi** [rosskhanas/react-qr-code#121]
Hey!

Importing the library leads to the following error:
```
./node_modules/react-qr-code/lib/components/QRCodeSurface/index.js
Module not found: Can't resolve 'react-art-svg-renderer' in '/Users/***/github/***/***/node_modules/react-qr-code/lib/components/QRCodeSurface'
```

This is probably related to the refactoring you recently made, where you removed `react-art-svg-renderer`. In the node_modules folder, the file indeed has the following line:

`var _reactArtSvgRenderer = require('react-art-svg-renderer');`

I should have the newest version, at least when looking from `yarn.lock`:
```
react-qr-code@^1.0.1:
  version "1.0.1"
  resolved "https://registry.yarnpkg.com/react-qr-code/-/react-qr-code-1.0.1.tgz#bff2966f5f7405aee902c4cc1d631dd74e1ea9b5"
  integrity sha512-+r1tkBelsU7HNfl3GVlB9XZyPVu/5KFa7iKzSMLL3WQymri10RHgG/aLH6VMfb78Ll24BwcMX2KYbIPN/kpKgQ==
  dependencies:
    "@react-native-community/art" "^1.2.0"
    prop-types "^15.7.2"
    qr.js "0.0.0"
```

Did you maybe forget to build the newest code before deploying to npm, and therefore it only included the changed to package.json but not the code? :D

**Comment by rosskhanas** [comment]
@lakaniemi sorry, I guess forgot to build the library before publishing to npm. I just published a fix in `1.0.2`.

**Comment by lakaniemi** [comment]
No problem, I just installed the missing package in the meantime, now after the new version it works perfectly without. Thanks for the package! :)
